tests/acceptance/features/bootstrap/SpacesContext.php

```diff
@@ -2911,4 +2962,44 @@ public function forUserSpaceShouldContainLinks(
 			Assert::assertEmpty($responseArray, __METHOD__ . ' Response should be empty');
 		}
 	}
+	   
+	/**
+	 * @When /^user "([^"]*)" gets the following properties of (?:file|folder|entry|resource) "([^"]*)" inside space "([^"]*)" using the WebDAV API$/
+	 *
+	 * @param string    $user
+	 * @param string    $resourceName
+	 * @param string    $spaceName
+	 * @param TableNode|null $propertiesTable
+	 *
+	 * @return void
+	 *
+	 * @throws Exception|GuzzleException
+	 */
+	public function userGetsTheFollowingPropertiesOfFileInsideSpaceUsingTheWebdavApi($user, $resourceName, $spaceName, TableNode $propertiesTable):void {
+		$space = $this->getSpaceByName($user, $spaceName);
+		$properties = null;
+		$fullUrl = $this->baseUrl . $this->davSpacesUrl . $space["id"] . $resourceName;
+		$this->featureContext->verifyTableNodeColumns($propertiesTable, ["propertyName"]);
+		$this->featureContext->verifyTableNodeColumnsCount($propertiesTable, 1);
+		if ($propertiesTable instanceof TableNode) {
+			foreach ($propertiesTable->getColumnsHash() as $row) {
+				$properties[] = $row["propertyName"];
+			}
+		}
+		$this->getPropertiesOfAResource($properties, $fullUrl, $user);
+	}
+
+	/**
+	 * @Then /^as user "([^"]*)" (?:file|folder|entry|resource) "([^"]*)" inside space "([^"]*)" should contain a property "([^"]*)" with value "([^"]*)"$/
+	 */
```

Comment:
Seems Parameters information is missing for this step.

Reply:
Forgot to add. Thanks! will be adding in the next commit!